Gemfile

```diff
@@ -10,8 +10,8 @@ group :test, :development do
   gem 'rspec', '~> 3.1'
 
   unless ENV['NO_ACTIVERECORD']
-    gem 'activerecord', '>= 3.2.3', '< 4.3.0'
-    gem 'activerecord-oracle_enhanced-adapter', '>= 1.4.1', '< 1.7.0'
+    gem 'activerecord', '>= 3.2.3', '< 5.1.0'
```

Comment:
travis.yml has a feature called [Build Matrix](https://docs.travis-ci.com/user/customizing-the-build#Build-Matrix) and I've found sprockets-rails repository is using this feature https://github.com/rails/sprockets-rails/tree/master/gemfiles